src/disemvowling.js

```diff
@@ -0,0 +1,3 @@
+export default function disemvowling(sentence){
+  return sentence.split('').filter((character)=>!character.match(/[aeiou ]/)).join('')
+}
```

Comment:
This does not remove uppercase vowels. You can add them to the regex or `.toLowercase()` the string first.